scenes/enemy-call-button/enemy-call-button.gd

```diff
@@ -0,0 +1,127 @@
+tool
+class_name EnemyCallButton
+extends Area2D
+
+signal enemy_start_timeout(enemy_call_button)
```

Comment:
The name it's okay, but it wasn't clear at the beginning for me. I had to think about it for a couple of minutes xD For me it's okay also as it is right now, in the future we could change it if others find it confusing.